pkg/spanconfig/spanconfigkvsubscriber/kvsubscriber.go

```diff
@@ -99,6 +109,23 @@ type KVSubscriber struct {
 
 var _ spanconfig.KVSubscriber = &KVSubscriber{}
 
+// KVSubscriberMetrics are the metrics associated with an instance of the
```

Comment:
Do these need to be exported? These symbols aren't accessible outside this package. They can also drop the KVSubscriber prefix. If we wanted a pkg/spanconfig wide metrics type to house all things, that could worth exporting (akin to our testing knobs), but we're not doing that here.

Reply:
Nope they don't, unexported.

Reply:
Acutally, I don't totally get why but the metric field has to be exported otherwise it doesn't showup in the generated chart catalog 🤔

Reply:
Hm, probably some go reflection thing we're using somewhere. Meh.